spec/features/new_rides_form_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+feature 'Admin adds new ride', %Q{
+  As an admin
+  I want to create a new ride
+  So that I can add a new ride to the Index page
+} do
```

Comment:
while we included user stories during SCs, generally you shouldnt do that in your more formal tests